src/timeline/media/css/main.css

```diff
@@ -144,10 +144,18 @@ img {
 .snapping-line.ng-hide-add.ng-hide-add-active { opacity:0.0; }
 
 /* Scrollbar style */
-::-webkit-scrollbar {width: 6px; height: 6px; background: #000000; }
-::-webkit-scrollbar-thumb { background-color: #4b92ad; -webkit-border-radius: 1ex; }
+::-webkit-scrollbar {width: 0.9em; height: 0.9em; background: #000000; }
```

Comment:
>... edges of the scrollbar track won't show black corners outside the `border-radius` area ...
It shouldn't. This is intentional styling, not a bug.

Reply:
It does, if you set the scrollbar background to black _without_ a `border-radius`:

![image](https://user-images.githubusercontent.com/538020/60817867-7f8c0980-a16a-11e9-92bd-7dcf7501ae03.png)

Those little black triangles at the corners of the rounded scrollbar track, are what I'm referring to. (But yay, no white box!)

Setting either `background:none` or `border-radius:0.45em` on the styling for the main `::-webkit-scrollbar` will get rid of those, and let the shape match the rounding of the `::-webkit-scrollbar-track` area. 

(This is with `background: none`, which I think is simpler. But it'll be identical either way):
![image](https://user-images.githubusercontent.com/538020/60818193-28d2ff80-a16b-11e9-9017-e25ef9e8aa72.png)

Reply:
(And like I said, that bug's a carryover from the old code. It's always been like that. It's just more noticeable now, with the larger scrollbars, than when the black area was only one pixel.)

Reply:
Scrollbars is way weird objects to render (and style), so not a bug. When you try to write your own scroll element you will understand why. Rectangular groove is quite normal. I just think that rounded - it looks better. I simply forgot to add corresponding changes to this PR (to smooth the corners of the scrollbar's track, by making them transparent). And thus, my screenshot shows normal behavior, while the track of the scrollbar still has rectangular background. I will update this. Actually, the only reason why background color appeared in the  `-webkit-scrollbar-track` - is because I removed it from the `-webkit-scrollbar`.

Reply:
>  When you try to write your own scroll element you will understand why. 

I look forward to having that novel experience that I've had literally dozens of times in the past. 

> so not a bug.

We clearly have different definitions of bug. I consider _this_ a styling bug:

![image](https://user-images.githubusercontent.com/538020/60907625-05ca4d80-a248-11e9-8628-fa09a832ad0e.png)

It's an easily-fixable one, and as you say it's now fixed in the updated code. :+1: